changelog.txt

```diff
@@ -8,3 +8,5 @@ Thanks for upgrading to the latest version of the ALKS CLI!
 Have feedback? https://github.com/Cox-Automotive/ALKS-CLI/issues
 
 ☁☁☁☁☁☁  Happy Clouding! ☁☁☁☁☁☁
+
+* adds optional JSON output for `alks developer accounts` command
```

Comment:
The automation can autofix the date at the top of this file but this bullet point should go up on line 6 and replace the one that's there for the automation to work properly